microservices/payment-stripe/src/services/payment-gateway/stripe.ts

```diff
@@ -1490,6 +1488,93 @@ class Stripe extends Abstract {
     };
   }
 
+  /**
+   * Returns payout method data
+   */
+  protected async getPayoutMethodAllowances(
+    userId: string,
+    payoutMethod?: IPayoutMethod,
+  ): Promise<{ externalAccountId: string; isInstantPayoutAllowed: boolean } | undefined> {
+    const externalAccountNotFoundError = messages.getNotFoundMessage(
```

Comment:
move this const to 1526 line

Reply:
I won't be able to access this variable outside of the "if block" if I transfer this message inside the if block.